recipes/python-symengine/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.0.git" %}
```

Comment:
Could we please have [PEP440]( https://www.python.org/dev/peps/pep-0440/ ) compliant version number? Perhaps changing `0.1.0.git` to `0.1.0.dev0`.